src/content/content_instance.cc

```diff
@@ -216,6 +217,42 @@ bool ContentInstance::HandleUpdateRequest(const picojson::value& msg) {
     }
   }
 
+  if (msg.contains(STR_ORIENTATION)) {
+    image_meta_h image;
+    if (media_info_get_image(handle, &image) == MEDIA_CONTENT_ERROR_NONE) {
+      std::string o = msg.get(STR_ORIENTATION).to_str();
```

Comment:
@pozdnyakov The .to_str() is needed because the msg.get(STR_ORIENTATION) return "picojson::value" which cannot revert to std::string implicitly.
I've updated "o" to "msg_orientation" for better reading.